### PR TITLE
Made autograde module be importable from labs + cleaned up logs

### PR DESCRIPTION
**Pull request by icanberk**
(ie. buflab can call sendJob now)

### CODE REVIEW

File: app/controllers/assessment/autograde.rb [hunk 1]
```diff
@@ -228,4 +235,83 @@ def autograde?(submission)
     not (submission.nil? or submission.handinFile().nil?)
   end
 
+  def extend_config_module
```
Comment (icanberk) [inline app/controllers/assessment/autograde.rb]: This code is repeated in submissions_controller and assessments_controller but I haven't had the chance to clean it up yet. I think it'll work if we just remove that method from but no time to test.

**Comment by dlbucci** [comment]
Should we really be allowing this?  It seems like it's a step backwards in our attempt to remove modules...

**Comment by icanberk** [comment]
I'm not sure if it's a step backwards. We just allow custom rb files send jobs to Tango. Although I think overwriting autogradeDone is pretty bad. Anyhow, we really need this for buflab coming out on Thursday...

**Comment by icanberk** [comment]
We need to rewrite buflab with a better logic in the future, but Dave didn't want to take the risk at this point.

**Comment by dlbucci** [comment]
Yeah, the fact that we need to rewrite it makes me think this is a step back, or at least a side step, but I understand.

**Pull request merged**